tools/IoTEdgeDevOps/DevOpsLib/BuildManagement.cs

```diff
@@ -21,25 +21,33 @@ public BuildManagement(DevOpsAccessSetting accessSetting)
             this.accessSetting = accessSetting;
         }
 
-        public async Task<VstsBuild> GetLatestBuildAsync(int buildDefinitionId, string branchName)
+        public async Task<IDictionary<int, VstsBuild>> GetLatestBuildsAsync(int[] buildDefinitionIds, string branchName)
         {
-            string requestPath = string.Format(LatestBuildPathSegmentFormat, this.accessSetting.Organization, this.accessSetting.Project, buildDefinitionId);
+            ValidationUtil.ThrowIfNulOrEmptySet(buildDefinitionIds, nameof(buildDefinitionIds));
```

Comment:
In IoTEdgeDevOps solution, I didn't reference anything from IoTEdge solution.  We can discuss to bring Preconditions to IoTEdgeDevOps.  Also I refactor to use HashSet for buildDefinitionIds argument to prevent duplication.